components/json_page/index.coffee

```diff
@@ -30,13 +30,13 @@ module.exports = class JSONPage
         .end (err, res) =>
```

Comment:
So based on http://visionmedia.github.io/superagent/#error-handling it looks like handling `res.error` is actually valid. I updated the code to be defensive about `res.error` and fall back to the `err.response`.